locale/ar/docs/guides/anatomy-of-an-http-transaction.md

```diff
@@ -12,7 +12,7 @@ layout: docs.hbs
 
 ## إنشاء الخادم
 
-أي تطبيق خادم الويب node وفي نقطة ما لإنشاء كائن خادم الويب، يتم ذلك
+أي تطبيق خادم الويب node.js وفي نقطة ما لإنشاء كائن خادم الويب، يتم ذلك
```

Comment:
@3imed-jaberi `node.js` is as wrong as `Node`. As per our branding guidelines, `node` is allowed when referring to the executable, and `Node.js` is to be used for the product in all other contexts. Neither `Node` nor `node.js` are correct.

Reply:
typo beceause i use change multiple !